include/mata/nfa/delta.hh

```diff
@@ -106,16 +106,17 @@ public:
     StatePost& operator=(StatePost&&) = default;
     using super::insert;
     using super::reserve;
-    using super::remove;
     using super::empty, super::size;
     using super::ToVector;
-    using super::erase;
```

Comment:
Because after renaming the remove to erase, erase became overloaded, and this line did not work. We will rewrite all those "using" things eventualy anyway.

Reply:
I believe that it should work, even if some changes are necessary. But I agree that we can refactor this later when we rewrite `StatePost` to have a member `symbol_posts` of type `std::vector<SymbolPost>`.